sdks/python/setup.py

```diff
@@ -282,11 +282,20 @@ def get_portability_package_data():
             'pyyaml>=3.12,<7.0.0',
             'requests_mock>=1.7,<2.0',
             'tenacity>=5.0.2,<6.0',
+            'onnxruntime==1.13.1',
+            'Pillow>=9.0.0',
             'pytest>=7.1.2,<8.0',
             'pytest-xdist>=2.5.0,<3',
             'pytest-timeout>=2.1.0,<3',
             'scikit-learn>=0.20.0',
+            'skl2onnx==1.13',
             'sqlalchemy>=1.3,<2.0',
+            'tensorflow>=2.11.0,<3',
```

Comment:
Do we want to add these in the test? 

Initially, there was a disagreement of adding ML libraries to the apache_beam[test], as it could install lot of unncessary dependencies and also big in size(tensorflow, torch) on to the user machine. 

Also, I am removing tensorflow from the python/containers/base_image_requirements_manual.txt since it is blocking the upgrade of protobuf.

Reply:
I am in favor of adding these dependencies to the respective gradle/tox tasks.